framework/src/org/checkerframework/framework/util/defaults/QualifierDefaults.java

```diff
@@ -500,6 +500,31 @@ private DefaultSet defaultsAt(final Element elt) {
         }
     }
 
+    public boolean applyUnannotatedDefaults(final Element annotationScope) {
+        boolean annotatedForThisChecker = isElementAnnotatedForThisChecker(annotationScope);
+
+        if (unannotatedDefaults.size() > 0) {
+            // TODO: I would expect this:
+            //   atypeFactory.isFromByteCode(annotationScope)) {
```

Comment:
In general, it's not clear to me what isFromByteCode current behavior is verse expect behavior.   Should we open an issue for this rather than document this in a todo (or code review)?